index.bs

```diff
@@ -7641,7 +7641,7 @@ To prevent this, a {{MediaElementAudioSourceNode}} MUST output
 {{HTMLMediaElement}} if it has been created using an
 {{HTMLMediaElement}} for which the execution of the
 <a href="https://fetch.spec.whatwg.org/#fetching">fetch
-algorithm</a> labeled the resource as
+algorithm</a> [[!FETCH]] labeled the resource as
```

Comment:
Could the link to the fetch algorithm be completely replaced by this (`[[!FETCH]]`)?

Reply:
I don't think so; the bibliographic reference is to the whole Fetch spec while the inline link is to the specific section cited.